### PR TITLE
7 BatchReportService interface

### LINKED ISSUE #7: BatchReportService interface
**Description of the feature to be developed:**
Make the interface "BatchReportService" and add the abstract methods found in the class diagram to it.

### PR DESCRIPTION
**Pull request by Teyson**
Closes #7

## Description
Added and populated the interface IBatchReportService

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests pass.
- [x] Sonar check has passed
- [x] No codesmells or bugs

## Other Relevant Information
There are no relevant tests to make.

### CODE REVIEW

**Review by Wylardt — APPROVED** [review]

**Review by Kaszz — APPROVED** [review]

**Review by Lawst96 — APPROVED** [review]
:shipit:

File: BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IBatchReportService.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.brewmes.common.services;
+
+public interface IBatchReportService {
+    public String prepareBatchReportService(int id);
```
Comment (dottore1) [inline BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IBatchReportService.java:4]: Bør denne tage String id i stedet?

Comment (Lawst96) [reply BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IBatchReportService.java:4]: Ja, det er et hash.

Comment (Lawst96) [reply BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IBatchReportService.java:4]:
```suggestion
    public String prepareBatchReportService(String id);
```

Comment (Teyson) [reply BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IBatchReportService.java]: Så er det fikset, tjek venligst igen <3

**Review by dottore1 — APPROVED** [review]

**Review by Kechr — APPROVED** [review]
:shipit:

**Pull request merged**